docs/generate_module_docs.py

```diff
@@ -0,0 +1,1269 @@
+    """Extract element rows from a complexType definition."""
+    rows: List[Dict[str, str]] = []
+    for child in complex_type:
+        if child.tag in {f"{XS_NS}sequence", f"{XS_NS}choice"}:
+            rows.extend(extract_elements_from_parent(child))
+    return rows
+
+
+def extract_model_mappings(schema_root: ET.Element) -> List[Dict[str, str]]:
+    """Read mappings from the amcr element choice into Model -> ComplexType rows."""
+    ns = {"xs": "http://www.w3.org/2001/XMLSchema"}
+    mappings: List[Dict[str, str]] = []
+
+    amcr_element = schema_root.find("xs:element[@name='amcr']", ns)
+    if amcr_element is None:
+        return mappings
+
+    choice = amcr_element.find(".//xs:choice", ns)
+    if choice is None:
+        return mappings
+
+    for element in choice:
+        if element.tag != f"{XS_NS}element":
+            continue
+        comment = get_following_comment(choice, element)
+        value_map, _ = parse_comment_values(comment)
+        model_name = value_map or element.attrib.get("name", "")
+        mappings.append({"model": model_name, "type": element.attrib.get("type", "")})
+
+    return mappings
+
+
+def generate_export_structure_rst() -> bool:
+    """Generate docs/source/05_integrace/export_structure.rst from amcr.xsd."""
+    xsd_file = project_root / "webclient/xml_generator/definitions/amcr.xsd"
+    output_file = docs_dir / "source/05_integrace/export_structure.rst"
+
+    print("\n  Generating export structure documentation")
+    print(f"  Source: {xsd_file}")
+    print(f"  Output: {output_file}")
+
+    if not xsd_file.exists():
+        print("  ⊝ XSD schema not found")
+        return False
+
+    try:
+        parser = ET.XMLParser(target=ET.TreeBuilder(insert_comments=True))
+        tree = ET.parse(xsd_file, parser=parser)
+    except Exception as exc:
+        print(f"  ✗ Failed to parse XSD: {exc}")
+        return False
+
+    schema_root = tree.getroot()
+    model_mappings = extract_model_mappings(schema_root)
+
+    documented_complex_types: List[ET.Element] = []
+    generic_type_names: List[str] = []
+    generic_boundary_reached = False
+
+    # Respect the position of the "generic-types" comment to split processing
+    for child in list(schema_root):
+        if child.tag is ET.Comment and (child.text or "").strip() == "generic-types":
+            generic_boundary_reached = True
+            continue
+
+        if child.tag == f"{XS_NS}complexType":
+            if generic_boundary_reached:
+                name = child.attrib.get("name", "")
+                if name:
+                    generic_type_names.append(name)
+            else:
+                documented_complex_types.append(child)
+
+    rst_lines = [
+        "Struktura XML exportu",
+        "=====================",
+        "",
+        "Tento dokument popisuje strukturu XML exportu z databáze AMCR podle XML schématu verze 2.2.",
```

Comment:
Tady se na pevno vkládá informace o verzi. To bude do budoucna matoucí, protože to určitě zapomeneme měnit. Údaj lze ale načíst z elementu:
`<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:amcr="https://api.aiscr.cz/schema/amcr/2.2/" targetNamespace="https://api.aiscr.cz/schema/amcr/2.2/" elementFormDefault="qualified" attributeFormDefault="unqualified" version="2.2">`
(na konci je a bude verze vždy napsaná v atributu "version")